src/Equinor.ProCoSys.IPO.WebApi/Synchronization/BusReceiverService.cs

```diff
@@ -129,23 +134,10 @@ private void ProcessMcPkgEvent(string messageJson)
                 });
             _plantSetter.SetPlant(mcPkgEvent.Plant);
 
-            if (!string.IsNullOrWhiteSpace(mcPkgEvent.McPkgNoOld))

```

Comment:
I don't understand how we now handle moving a McPkg from a CommPkg to another CommPkg.